dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/DataSource.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.dolphinscheduler.dao.entity;
 
+import com.baomidou.mybatisplus.annotation.*;
```

Comment:
This should be avoided. cc @EricGao888

Reply:
@SbloodyS That's pretty wired. We have a step in Spotless to check and block wildcard imports. Usually CI would detect it and block this PR. The reason why it didn't block it may be our current setting for `ratchet`. https://github.com/apache/dolphinscheduler/blob/e1b55db72ad14ac8372c495a6b1398d686f02365/pom.xml#L660-L664

For details, see: #11412